sdcm/cluster_gce.py

```diff
@@ -300,6 +301,13 @@ def _create_instance(self, node_index, dc_idx, spot=False):
         # Name must start with a lowercase letter followed by up to 63
         # lowercase letters, numbers, or hyphens, and cannot end with a hyphen
         assert len(name) <= 63, "Max length of instance name is 63"
+        startup_script = cluster.Setup.get_startup_script()
+        if self.params.get("scylla_linux_distro", "") in ("ubuntu-bionic", "ubuntu-xenial"):
+            # we need to disable sshguard to prevent blocking connections from the builder
+            startup_script += dedent("""
```

Comment:
Can we disable sshguard as early as possible?
Currently it's after restarting sshd.